acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -20,21 +20,24 @@ const verbToRedirectLinkSuffix =  {
 };
 export default function init(element) {
   const widget = element;
-  let WIDGET_ENV = 'https://dev.acrobat.adobe.com/dc-hosted/2.37.2_1.165.0/dc-app-launcher.js';
+  let DC_WIDGET_VERSION_FALLBACK = '2.37.2_1.165.0';
+  let DC_GENERATE_CACHE_VERSION_FALLBACK = '1.165.0';
+  let DC_WIDGET_VERSION = document.querySelector('meta[name="dc-widget-version"]')?.getAttribute('content') || DC_WIDGET_VERSION_FALLBACK;
```

Comment:
providing a fallback means we won't notice a setup error.  Should we fail instead or log the error?

Reply:
...but do we know if lana logging will be available this early?

Reply:
we can capture the error and send it when lana loads 
we already do this in other places @peller